core/store/src/trie/trie_storage.rs

```diff
@@ -73,6 +75,14 @@ pub struct TrieCacheInner {
     shard_id: u32,
     /// Whether cache is used for view calls execution.
     is_view: bool,
+
+    // Store counteres here to avoid overhead of looking them up on hot paths.
```

Comment:
```suggestion
    // Counters tracking operations happening inside the shard cache. Stored here to avoid overhead of looking them up on hot paths.
```